src/target/riscv/riscv.c

```diff
@@ -462,11 +462,76 @@ static void trigger_from_breakpoint(struct trigger *trigger,
 	trigger->unique_id = breakpoint->unique_id;
 }
 
-static int maybe_add_trigger_t1(struct target *target,
-		struct trigger *trigger, uint64_t tdata1)
+static bool match_napot_checker(struct trigger *trigger, riscv_reg_t *tdata2)
```

Comment:
I would recommend a more descriptive name:

```suggestion
static bool can_use_napot_match(struct trigger *trigger, riscv_reg_t *tdata2)
```